public/apps/types.ts

```diff
@@ -21,3 +21,7 @@ export interface AppDependencies {
   navigation: AppPluginStartDependencies;
   params: AppMountParameters;
 }
+
+export interface SingleResourcePageDependencies extends AppDependencies {
```

Comment:
Looks like BreadcrumbsPageDependencies is more close to what this interface performs.

Reply:
Good idea.